gflownet/envs/base.py

```diff
@@ -58,10 +59,13 @@ def __init__(
             self.oracle = self.proxy
         else:
             self.oracle = oracle
-        if self.oracle.higher_is_better:
-            self.proxy_factor = 1.0
-        else:
-            self.proxy_factor = -1.0
+            # higher_is_better is dependant on proxy
+            # but proxy is set after env is iniatiliased
+            # TODO: need to think about how to do iniatilise higherIsBetter
```

Comment:
I don't understand the problem. The proxy is initialized (in `main.py`) before the environment is actually passed as an argument to the environment. So what's the problem of taking `higher_is_better` from the proxy?

Reply:
In the active learning code, the proxy (acquisition function) is initialized after the env as the acquisition function is dependant on the regressor (MLP), which is dependant on the env. I've added a `set_proxy()` function to take care of this